### PR TITLE
Feat: Refactor Forms Update and Delete Routes

### PR DESCRIPTION
**Pull request by JesusGo115**
### Description

Removed the update/delete routes for survey.routes and added update/delete routes for Forms

### Issue Link

[Jira Card Here](https://cascarita.atlassian.net/browse/C1-106)

### Checklist

- [x] I have tested the changes locally by running `npm run test`
- [x] I have autoformatted the code by running `npm run eslint`

### CODE REVIEW

File: server/controllers/form.controller.js [hunk 1]
```diff
@@ -99,6 +99,99 @@ const FormController = {
...
+        await FormMongo.updateOne(
+          { _id: form_id },
+          {
+            $set: mongoUpdateData,
+            $currentDate: { updatedAt: true },
+          },
+        );
+      }
+
+      res.json(formResponse);
```
Comment (saulRM10) [inline server/controllers/form.controller.js:166]: you should return a status code for a successful update

Comment (JesusGo115) [reply server/controllers/form.controller.js]: Added!

**Review by abanuelo — APPROVED** [review]
Looks great thanks for making those changes! You are going to get you laptop now :)

**Review by abanuelo — APPROVED** [review]
LGTM thanks for making these changes!

**Pull request merged**